alibi/explainers/integrated_gradients.py

```diff
@@ -663,6 +713,7 @@ class IntegratedGradients(Explainer):
     def __init__(self,
                  model: tf.keras.Model,
                  layer: Optional[tf.keras.layers.Layer] = None,
+                 target_fn: Optional[Callable] = None,
```

Comment:
could we have `partial(np.argmax, axis=1)` as default instead?

Reply:
Unfortunately not as `target_fn` depends on the output shape of the predictions which can be anything, so it's up to the user to specify, we cannot safely assume a default here.